shake-bench/src/Development/Benchmark/Rules.hs

```diff
@@ -250,22 +251,16 @@ benchRules build benchResource MkBenchRules{..} = do
                 AddPath [takeDirectory ghcPath, "."] []
               ]
               BenchProject {..}
-          liftIO $ renameFile "ghcide.hp" $ dropFileName outcsv </> dropExtension exp <.> "hp"
+          liftIO $ renameFile "ghcide.hp" outHp
 
         -- extend csv output with allocation data
         csvContents <- liftIO $ lines <$> readFile outcsv
         let header = head csvContents
             results = tail csvContents
             header' = header <> ", maxResidency, allocatedBytes"
         results' <- forM results $ \row -> do
-            -- assume that the gcStats file can be guessed from the row id
-            -- assume that the row id is the first column
-            let id = takeWhile (/= ',') row
-            let gcStatsPath = dropFileName outcsv </> escapeSpaces id <.> "benchmark-gcStats"
-            (maxResidency, allocations) <- liftIO $
-                ifM (IO.doesFileExist gcStatsPath)
```

Comment:
Why remove this check?

Reply:
In a previous change, I moved the logic that generates this file from ghcide-bench to shake-bench, so there is no more guessing involved